sparkle/src/components/Avatar.tsx

```diff
@@ -96,6 +99,9 @@ export function Avatar({
   };
 
   const getTextVariant = (name: string) => {
+    if (/^\d+$/.test(name)) {
```

Comment:
I don't understand this part, could you give me some more context? 🙏🏻

Reply:
If it's a number, it will give a grey color

Reply:
I would remove that because that's really not easy to discover it! Could we handle the +2 without a fake avatar component to prevent this hack? 🙏🏻